### PR TITLE
Update free-courses-ar.md

### PR DESCRIPTION
**Pull request by Adamkaram**
## What does this PR do?
Adding  linux system course

### Description
all about linux system from zero and i highly recommend it for anyone need to know how to deal with linux system and configure it 

### Why is this valuable (or not)?
i have learn a lot from it 

### How do we know it's really free?
it's youtube playlist 

### For book lists, is it a book? For course lists, is it a course? etc.
course

## Checklist:
- [ ✓] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ✓] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ✓] Include author(s) and platform where appropriate.
- [ ✓] Put lists in alphabetical order, correct spacing.
- [ ✓] Add needed indications (PDF, access notes, under construction).
- [ ✓] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. Add to the Operating Systems category instead.

**Comment by Adamkaram** [comment]
> It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. Add to the Operating Systems category instead.

i will add more resource on that cat

**Comment by Adamkaram** [comment]
> It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. Add to the Operating Systems category instead.

done i've add 4 resource

**Review by eshellman — CHANGES_REQUESTED** [review]
Please use the "Operating Systems' Category

File: courses/free-courses-ar.md [hunk 1]
```diff
@@ -354,6 +355,14 @@
 * [Vue.js Router Tutorial&rlm;](https://www.youtube.com/playlist?list=PLfDx4cQoUNObtw2FtE4_eX_k9yCf-Fcd3) - Algorithm Academy
 
 
+### linux
+
+* [Linux System&rlm;](https://www.youtube.com/playlist?list=PL8pYI62gCNsWTppELEUCpforC4avEiLox) - anahr
+* [Operating Systems - أنظمة التشغيل](https://www.youtube.com/playlist?list=PLxIvc-MGOs6ib0oK1z9C46DeKd9rRcSMY) - Dr.Ahmed Hagag
```
[the remark this answers is not in the document]
Comment (Adamkaram) [reply courses/free-courses-ar.md]: i have been move three of them to OS sould i leave linux with one resource?

**Comment by eshellman** [comment]
linux is an operating system

**Review by Adamkaram — COMMENTED** [review]
omit linux cato

**Pull request closed without merging**